pkg/products/rhsso/reconciler.go

```diff
@@ -261,7 +261,7 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 	host := r.Config.GetHost()
 	if host == "" {
 		r.Log.Info("URL for Keycloak not yet available")
-		return integreatlyv1alpha1.PhaseAwaitingComponents, fmt.Errorf("Host for Keycloak not yet available")
+		return integreatlyv1alpha1.PhaseAwaitingComponents, nil
```

Comment:
Same as before, we should still pass out the error message and let what ever is receiving the error log it or not.